HUD/GetScaleformMovieCursorSelection.md

```diff
@@ -6,10 +6,20 @@ aliases: ["0x632B2940C67F4EA9"]
  
 ```c
 // 0x632B2940C67F4EA9
-BOOL _GET_SCALEFORM_MOVIE_CURSOR_SELECTION(int scaleformHandle, cs_type(Any*) bool* received, cs_type(Any*) int* selectionType, cs_type(Any*) int* slotIndex, int* p4);
+BOOL _GET_SCALEFORM_MOVIE_CURSOR_SELECTION(int scaleformHandle, cs_type(Any*) bool* received, cs_type(Any*) int* selectionType, cs_type(Any*) int* context, int* slotIndex);
 ```
- 
-Gets mouse selection data from scaleforms with mouse support. Must be checked every frame
+  
+## Parameters
+* **scaleformHandle**: Handle of the scaleform
+* **selectionType**: The type of MouseEvent specified above.
```

Comment:
actually, double checking everything again, seems i missed one more thing. All description lines must be directly below the top codeblock, and above the Parameters section. Currently Lines 19 and 23-45 will NOT make it into the description. Sorry about that.

Reply:
ok latest commit should be fine? I'm learning while doing it.. i apologize for the long review.

Reply:
ooooh ok!

Reply:
i still think that changing this Native wouldn't be a big problem for backward compatibility as for C# it would fix its behavour and for Lua/Js it would still work as the same.